<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="14911" started-at="2016-05-19T11:44:47Z" finished-at="2016-05-19T11:45:02Z">
    <groups>
      <group name="p1">
        <method signature="LoginTest.testLoginMainPage(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.testngtutorial.MyTestNGTutorial.LoginTest@cc285f4]" name="testLoginMainPage" class="com.testngtutorial.MyTestNGTutorial.LoginTest"/>
      </group> <!-- p1 -->
    </groups>
    <test name="Default test" duration-ms="14911" started-at="2016-05-19T11:44:47Z" finished-at="2016-05-19T11:45:02Z">
      <class name="com.testngtutorial.MyTestNGTutorial.LoginTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.testngtutorial.MyTestNGTutorial.LoginTest@cc285f4]" name="setup" is-config="true" duration-ms="5689" started-at="2016-05-19T14:44:47Z" finished-at="2016-05-19T14:44:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="testLoginMainPage(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.testngtutorial.MyTestNGTutorial.LoginTest@cc285f4]" name="testLoginMainPage" duration-ms="7093" started-at="2016-05-19T14:44:53Z" data-provider="login" finished-at="2016-05-19T14:45:00Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[k.i.work@mail.ru]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Feamarth2Facebook]]>
              </value>
            </param>
            <param index="2">
              <value is-null="true"/>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLoginMainPage -->
        <test-method status="FAIL" signature="testLoginMainPage(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.testngtutorial.MyTestNGTutorial.LoginTest@cc285f4]" name="testLoginMainPage" duration-ms="2080" started-at="2016-05-19T14:45:00Z" data-provider="login" finished-at="2016-05-19T14:45:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Theemail]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Valid message displayed expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Valid message displayed expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.testngtutorial.MyTestNGTutorial.LoginTest.testLoginMainPage(LoginTest.java:65)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLoginMainPage -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.testngtutorial.MyTestNGTutorial.LoginTest@cc285f4]" name="teardown" is-config="true" duration-ms="21" started-at="2016-05-19T14:45:02Z" finished-at="2016-05-19T14:45:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.testngtutorial.MyTestNGTutorial.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
